jupyter-book/overfit/overfit_take_away.md

```diff
@@ -6,23 +6,27 @@
   **noise** in the data, and the **high flexibility** of common machine learning
   models.
 
-- **Underfitting** happens when the learned prediction functions make
-  **systematic errors**. This can be caused by a choice of the model family and
-  parameters that leads to a **lack of flexibility** to capture the repeatable
+- **Underfitting** happens when the learnt prediction functions suffer from
```

Comment:
learned vs learnt: https://www.grammarly.com/blog/learned-learnt/

learned didn't shock me at all, I think we are learning more towards US English in this course.

Same thing as above though, let's keep it as it is, higher priorities ...

Reply:
I agree, and while both are valid, I wrote like I would talk.

Speaking out loud, I think ``learnt prediction`` sounds better than ``learned prediction``.
Might just be me :)